index.js

```diff
@@ -138,6 +138,32 @@ function getSequelizeTypeFromJoi(dialect, type, rules) {
     }
 }
 
+/**
+ * Checks if default value is allowed for the specified dialect and Sequilize type
+ * @method getSequelizeTypeFromJoi
+ * @param  {String}          dialect Underlying system that we're writing to
+ * @param  {String}          type    Sequelize type
+ * @return {Boolean}                 true if default value is allowed. false otherwise.
+ */
+function isDefaultValueAllowedForSequilizeType(dialect, type) {
```

Comment:
probably don't need such a long func name
```suggestion
function isDefaultValueAllowed(dialect, type) {
```